src/plugins/intel_cpu/src/nodes/executors/shl/shl_eltwise.hpp

```diff
@@ -0,0 +1,50 @@
+#include "shl.hpp"
```

Comment:
Could you please add the following lines at the begin of the file?
```
// Copyright (C) 2024 Intel Corporation
// SPDX-License-Identifier: Apache-2.0
//
```

Also need to add `#pragma once` since it's header file